src/components/Draft.js

```diff
@@ -61,21 +53,20 @@ function Page() {
     messagePropStates()
   }, [propStates])
 
+  // useEffect(() => {
+  // setItem(
+  //   JSX_STRING
+  //   // reactElementToJSXString(<SelectedComponent {...deserializeAll(propStates)} />)
+  // )
+  // }, [propStates])
```

Comment:
Note that we are keeping the commented out code here for our work on #22 and #19